plugins/renamerOnUpdate/README.md

```diff
@@ -1,19 +1,19 @@
 
 # SQLITE Renamer for Stash
-Using metadata from your stash to rename your file.
+Using metadata from your Stash to rename your files.
 
 ## Requirement
 - Stash
 - Python 3+ (Tested on Python v3.9.1 64bit, Win10)
 - Request Module (https://pypi.org/project/requests/)
-- Windows 10 ? (No idea if this work for all OS)
+- Tested on Windows 10 and Synology/docker (No idea if this work for all OS)
 
 ## Installation
 
 - Download the whole folder 'renamerOnUpdate' (config.py, log.py, renamerOnUpdate.py/.yml)
 - Place it in your **plugins** folder (where the `config.yml` is)
 - Reload plugins (Settings > Plugins)
-- renamerOnUpdate should appear. 
+- renamerOnUpdate should appears 
```

Comment:
`renamerOnUpdate should appear` sounded ok or did you want to replace  with `renamerOnUpdate appears` ? Either the `should` or the `s` from appears has to be removed